integration-tests/suites/k8s/config_reload.go

```diff
@@ -4,27 +4,24 @@ import (
 	"github.com/stackrox/collector/integration-tests/pkg/assert"
 	"github.com/stackrox/collector/integration-tests/pkg/collector"
 	"github.com/stackrox/collector/integration-tests/pkg/log"
+	"github.com/stackrox/collector/integration-tests/pkg/types"
 
 	coreV1 "k8s.io/api/core/v1"
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const (
-	EXT_IP_ENABLE = `
-networking:
-  externalIps:
-    enable: true
-`
-
-	EXT_IP_DISABLE = `
-networking:
-  externalIps:
-    enable: false
-`
+var (
+	EXT_IP_ENABLE  string
+	EXT_IP_DISABLE string
 
 	CONFIG_MAP_NAME = "collector-config"
 )
 
+func init() {
+	EXT_IP_ENABLE, _ = types.GetRuntimeConfigEnabledStr("ENABLED")
+	EXT_IP_DISABLE, _ = types.GetRuntimeConfigEnabledStr("DISABLED")
```

Comment:
We should handle these errors instead of ignoring them. Doesn't need to be fancy, we can just panic but it'll help if we accidentally break the runtime config part of the tests

Reply:
Done